src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/history.html

```diff
@@ -29,6 +29,7 @@
                             <pre>{{ value.message|linebreaksbr }}</pre>
                         {% endfor %}
                     </p>
+                    <a href="{% url 'plugins:sendmail:send' organizer=request.event.organizer.slug event=request.event.slug %}?from_log={{ log.id }}">{% trans "Send a new email based on this" %}</a>
```

Comment:
hi @raphaelm ,

I think it is more reasonable when we use `from_log={{ log.id }}` instead of `from_log={{ log.object_id }}`

After using id, the message can be copied now. Please review the code :)